src/platform/packages/shared/kbn-unified-data-table/src/hooks/use_row_height.ts

```diff
@@ -138,10 +136,18 @@ export const useRowHeight = ({
     [configRowHeight, consumer, key, onUpdateRowHeight, storage]
   );
 
+  useEffect(() => {
+    if (rowHeight === RowHeightMode.custom) {
+      setLineCountInput(rowHeightState && rowHeightState > 0 ? rowHeightState : configRowHeight);
+    }
+  }, [rowHeightState, configRowHeight, rowHeight]);
+
```

Comment:
We can't set input value within onChangeRowHeightLines because sometimes it comes from URL (hence useEffect watching `rowHeightState`). In case like this, above function is not triggered. If someone more experienced with Kibana has better idea, any insight is appreciated!

Reply:
Thanks, resolving then!